main.py

```diff
@@ -9,14 +9,17 @@
 MAX_READ: int = 16
 PORT: str = "/dev/ttyACM0"
 REFRESH_RATE: int = 1440
+	
+INITIAL_PERCENTAGE: int = 100
+SENSITIVE_PERCENTAGE: int = 50
 
 def start():
 	feather = mcu.MCUInterface(PORT,
                                baud: int = BAUD_RATE,
                                close_on_startup: bool = CLOSE_ON_STARTUP,
                                refresh_rate: int = REFRESH_RATE,
                                max_read: int = MAX_READ)
-	communications = comms.Communications(feather)
+	communications = comms.Communications(feather, SENSITIVE_PERCENTAGE, initial_percentage: int = INITIAL_PERCENTAGE)
```

Comment:
Just pass in `INITIAL_PERCENTAGE` instead of `initial_percentage: int = INITIAL_PERCENTAGE`.